src/Storageless/Http/ClientFingerprint/RemoteAddr.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PSR7Sessions\Storageless\Http\ClientFingerprint;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+use function array_key_exists;
+use function is_string;
+
+/** @immutable */
+final class RemoteAddr implements Source
+{
+    public const REQUEST_ATTRIBUTE_NAME = 'REMOTE_ADDR';
```

Comment:
There is a problem with relying on `REMOTE_ADDR`: it does not consider any proxies. If we assume that there's an upfront middleware that clears all this information, that's fine, but this `Source` is unreliable when the middleware is deployed independently, and put behind a reverse proxy.

Reply:
The only thing this library can do about that is documentation:
https://github.com/psr7-sessions/storageless/tree/9.0.x#session-hijacking-mitigation